### PR TITLE
Add usage examples to loss docstrings

### PR DESCRIPTION
**Pull request by mahi21tha**
This PR adds simple usage examples to loss docstrings to improve API documentation.

- Uses keras.ops for backend-agnostic examples
- No functional changes

### CODE REVIEW

File: keras/src/losses/losses.py [hunk 1]
```diff
@@ -73,6 +73,14 @@ class MeanSquaredError(LossFunctionWrapper):
             `"float32"` unless set to different value
             (via `keras.backend.set_floatx()`). If a `keras.DTypePolicy` is
             provided, then the `compute_dtype` will be utilized.
+
+    Examples:
+        >>> import keras
```
Comment (fchollet) [inline keras/src/losses/losses.py]: - Do not import keras (it's implicit)
- Unindent code example
- Add line break after "Examples:"

Comment (21mahitha) [reply keras/src/losses/losses.py]: ok done

File: keras/src/losses/losses.py [hunk 2]
```diff
@@ -73,6 +73,14 @@ class MeanSquaredError(LossFunctionWrapper):
             `"float32"` unless set to different value
             (via `keras.backend.set_floatx()`). If a `keras.DTypePolicy` is
             provided, then the `compute_dtype` will be utilized.
+
+    Examples:
+
+        >>> y_true = keras.ops.array([1.0, 0.0, 1.0])
+        >>> y_pred = keras.ops.array([0.9, 0.1, 0.8])
+        >>> loss = keras.losses.MeanSquaredError()
+        >>> loss(y_true, y_pred)
```
Comment (fchollet) [inline keras/src/losses/losses.py]: Also add the code output on the last line (since using `>>>`)

And unindent the lines (remove spaces before `>>>`)

File: requirements.txt [hunk 3]
```diff
@@ -6,7 +6,8 @@ tensorflow~=2.20.0;sys_platform == 'darwin'
 tf2onnx
 # This is the only version which works with TF 2.20.0.
 # TODO(#21914): Update this version when TF is updated.
-ai-edge-litert==1.3.0
+# ai-edge-litert==1.3.0  # not supported on Windows
+
```
Comment (hertschuh) [inline requirements.txt]: Remove extra blank line.

File: requirements.txt [hunk 4]
```diff
@@ -6,7 +6,8 @@ tensorflow~=2.20.0;sys_platform == 'darwin'
 tf2onnx
 # This is the only version which works with TF 2.20.0.
 # TODO(#21914): Update this version when TF is updated.
-ai-edge-litert==1.3.0
+# ai-edge-litert==1.3.0  # not supported on Windows
```
Comment (hertschuh) [inline requirements.txt]: Undo this file or change it to:
```
ai-edge-litert==1.3.0;sys_platform != 'win32'
```

File: keras/src/losses/losses.py [hunk 5]
```diff
@@ -114,6 +124,18 @@ class MeanAbsoluteError(LossFunctionWrapper):
             `"float32"` unless set to different value
             (via `keras.backend.set_floatx()`). If a `keras.DTypePolicy` is
             provided, then the `compute_dtype` will be utilized.
+
```
Comment (hertschuh) [inline keras/src/losses/losses.py:125]: Remove extra blank line.

File: keras/src/losses/losses.py [hunk 6]
```diff
@@ -114,6 +124,18 @@ class MeanAbsoluteError(LossFunctionWrapper):
...
+    Examples:
+
+    >>> y_true = keras.ops.array([1.0, 0.3, 1.0])
+    >>> y_pred = keras.ops.array([1.9, 0.3, 1.8])
+    >>> loss = keras.losses.MeanAbsoluteError()
+    >>> loss(y_true, y_pred)
+    0.5666667
+
+
+
```
Comment (hertschuh) [inline keras/src/losses/losses.py]: Remove these 3 blank lines.

File: keras/src/losses/losses.py [hunk 7]
```diff
@@ -155,6 +177,17 @@ class MeanAbsolutePercentageError(LossFunctionWrapper):
             `"float32"` unless set to different value
             (via `keras.backend.set_floatx()`). If a `keras.DTypePolicy` is
             provided, then the `compute_dtype` will be utilized.
+
```
Comment (hertschuh) [inline keras/src/losses/losses.py:174]: Remove extra blank line.

File: keras/src/losses/losses.py [hunk 8]
```diff
@@ -155,6 +177,17 @@ class MeanAbsolutePercentageError(LossFunctionWrapper):
...
+    Examples:
+
+    >>> y_true = keras.ops.array([100.0, 200.0, 300.0])
+    >>> y_pred = keras.ops.array([90.0, 210.0, 310.0])
+    >>> loss = keras.losses.MeanAbsolutePercentageError()
+    >>> loss(y_true, y_pred)
+    6.111111
+
+
```
Comment (hertschuh) [inline keras/src/losses/losses.py]: Remove these 2 blank lines.

File: keras/src/losses/losses.py [hunk 9]
```diff
@@ -73,6 +73,16 @@ class MeanSquaredError(LossFunctionWrapper):
...
+    Examples:
+
+    >>> y_true = keras.ops.array([1.0, 0.0, 1.0])
+    >>> y_pred = keras.ops.array([0.9, 0.1, 0.8])
+    >>> loss = keras.losses.MeanSquaredError()
+    >>> loss(y_true, y_pred)
+    0.02
+
+
```
Comment (hertschuh) [inline keras/src/losses/losses.py]: Remove these 2 blank lines.

**Review by hertschuh — COMMENTED** [review]
Thank you for the addition!

Some small tweaks

File: keras/src/losses/losses.py (hunk 9 above)
Comment (21mahitha) [reply keras/src/losses/losses.py]: sure done
